internal/config/config.go

```diff
@@ -8,28 +8,34 @@ import (
 	"github.com/vatsal278/TransactionManagementService/internal/model"
```

Comment:
Looks like all flows are yet to be covered. Can the remaining lines of code be covered?
https://app.codecov.io/gh/vatsal278/TransactionManagementService/pull/4#34d4dc28e4e4b8f9e871d6d68f62ef07-L1